ompi/mca/osc/ucx/osc_ucx.h

```diff
@@ -14,6 +14,11 @@
 
 #include "ompi/group/group.h"
 #include "ompi/communicator/communicator.h"
+#include "opal/mca/common/ucx/common_ucx.h"
+
+#define OSC_UCX_ASSERT MCA_COMMON_UCX_ASSERT
```

Comment:
why not just use MCA_COMMON_UCX_ASSERT in osc_ucx?
also, need to indent on column

Reply:
two reasons to use OSC macro:
1. it is shorter named
2. it may be configured separately per component in future - additional flexibility for free (own logging level, prefixes, etc)

indentation fixed